sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -346,25 +353,44 @@ public String getEntityPath() {
     }
 
     /**
-     * Receives a stream of {@link ServiceBusReceivedMessage messages} from the Service Bus entity.
+     * Receives a stream of {@link ServiceBusReceivedMessage messages} from the Service Bus entity and completes them
+     * when they are finished processing.
      *
      * @return A stream of messages from the Service Bus entity.
      */
     public Flux<ServiceBusReceivedMessage> receive() {
+        return receive(defaultReceiveOptions);
+    }
+
+    /**
+     * Receives a stream of {@link ServiceBusReceivedMessage messages} from the Service Bus entity.
+     *
+     * @return A stream of messages from the Service Bus entity.
```

Comment:
May be explaining here that  `To disable auto-renew, user have to set MaxAutoRenewDuration to zero.` And default is true